edgeworkers/examples/respond-from-edgeworkers/responseprovider/jsonp-wrapper/main.js

```diff
@@ -25,7 +25,8 @@ export function responseProvider(request) {
             controller.enqueue(str2uint8arr(")"));
         }
     });
-  
+    
+    const options = { 'Content-Type': ['application/json'] };
```

Comment:
Since this is used for a request, it would make more sense to use the [Accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) header with `'application/json'`

Also the [API for options on httpRequest](https://learn.akamai.com/en-us/webhelp/edgeworkers/edgeworkers-user-guide/GUID-D580F321-9124-4D2F-BB02-D8D81E88A461.html#GUID-D580F321-9124-4D2F-BB02-D8D81E88A461) specifies that you would need something more like:
`const options = { 'headers':  {'Accept': 'application/json'} };`

I think that'd probably be the best way to do things for an example -- it's likely in this case the options just weren't needed as it's an optional parameter and the param was just a vestige, but including an accept header would make it clearer for customers.

Reply:
Yes, including an accept header in options would make it clearer for customers. I have updated it as suggested by you.